test/e2e/http/actors.test.ts

```diff
@@ -79,6 +90,23 @@ describe('http/actors', () => {
     // await client.stop();
   });
 
+  describe('configuration', () => {
+    it('actor configuration endpoint should contain the correct parameters', async () => {
+      const res = await fetch(`http://${serverHost}:${serverPort}/dapr/config`)
+      expect(res.status).toBe(200);
+
+      const config = JSON.parse(await res.text())
+
+      expect(config.entities.length).toBe(8);      
+      expect(config.actorIdleTimeout).toBe('1h');
+      expect(config.actorScanInterval).toBe('30s');
+      expect(config.drainOngoingCallTimeout).toBe('1m');
+      expect(config.drainRebalancedActors).toBe(true);
+      expect(config.reentrancy.enabled).toBe(true);
```

Comment:
Should we have one test, where-in if we don't set few options (like we don't set `reentrance.enabled` to be `true`), then we are able to test their default values getting set?

Reply:
Default values won't be set from SDK. Runtime is responsible for that. I can add a test that confirms these values are omitted - but it will be the same as the existing unit test for that method.

Reply:
Yups, defaut values are set in runtime only. As it is an E2E test, so, I meant if we can somehow fetch Actors' configuration and test the expectations.